docs/courses/csintro3/orientation/logic.md

```diff
@@ -1,20 +1,55 @@
-# logic operators
+# Logic Operators
 
-table of blocks -> JavaScript equivalent
+One of the key topics covered with blocks was the idea of using logic to read the state of a program, and alter its behavior accordingly. As expected, everything that we did in blocks, we can do in JavaScript and we can actually do more.
 
-|     |     |
-| --- | --- |
-| a and b | a && b |
-| not a | !a |
+### If and else statements 
 
-etc
+Some of the key blocks we used for logic were the ``||logic:if||`` and ``||logic:if else||`` blocks.
 
-### If and else statements 
+To write these in JavaScript, we simply type
 
 ```typescript
-let num: number = 1;
-if (num == 1) console.log("num is one!");
-else console.log("num is not one!");
+if(condition) {
```

Comment:
I think that makes sense for 14x because they learn the syntax for functions before logic, but in this case it's the other way around so it's probably best to omit that detail. I think it will make things more confusing for them.

Reply:
They learn loops basically the same time they learn functions in 14x, which is the same time frame as this is set up (1-2 lessons between loops and functions, just opposite order in this case). The same rule follows for loops as logic, and they are distinct enough from functions that it's important for them to identify at some point.

It doesn't have to be explicitly taught that there's a space before the if condition, but it should be shown that way as it matches the style of the code written anywhere else that they might look at, the code that is generated, code that is formatted (right click and press format in Arcade editor) and general style guidelines. Omitting the space will consistently lead to inconsistent examples (see ``### Examples of using Comparisons and Boolean Operators`` below where you have the space before the parentheses vs earlier examples), which is something that would be likely to cause confusion / look unpolished.

Reply:
I misunderstood. I agree that there should be a space. I thought those were two separate thoughts and you were saying it is important to explicitly say here that if / for / etc. aren't functions.

Reply:
Ah yeah I'm indifferent on actually saying it (maybe when introducing anonymous functions to explain events give a quick hint box on it so they don't try and do weird stuff like pass an if to the function?)